.github/workflows/test_new_highway.yml

```
@@ -7,7 +7,9 @@
 
 name: Update branch with updated submodule on libjxl
 
-on: repository_dispatch
+on:
+  schedule:
+    - cron: '37 2 * * *' # Daily on 02:37
```

```suggestion
    - cron: '37 2 * * *' # Daily on 02:37 UTC
```
That is the time zone, right?

---

Correct.

---

Isn't it better to do something like 3:30AM? it's more comfortably away from the summer time change :P